tests/logger.config

```diff
@@ -23,7 +23,7 @@ STDOUT_LOG_LEVEL: DEBUG
 FILE_LOG_LEVEL: DEBUG
 
 // Print logs at this level and above to NETWORK, must be in all caps.
-NETWORK_LOG_LEVEL: INFO
+NETWORK_LOG_LEVEL: DEBUG
```

Comment:
Was this an intended production change? Pretty sure we decided on not sending `DEBUG` logs to Dawn?

Reply:
This is the test logger config. I made it DEBUG so I could actually see my debug statements print during the test. I think this should be the default for testing.

Reply:
Mm if we still do print output comparisons in tests then I hesitate to change this, @levincent06 do you know if any of our tests still use output comparisons?

Reply:
There are still tests that use output comparisons, specifically those where we test for output that doesn't get stored in shared memory. For example, see `tc_71_2.c` where we test that the student code properly has a traceback error (one divided by zero) We also have @0neDividedbyZer0's  (how fitting lol) regex addition for future output comparisons, if needed.

There's also `tc_71_1` which tests for start position through output, but that was an oversight and I'll add direct shared memory checking in another PR, specifically addressing issue #155 

If I'm understanding correctly, the issue with lowering the log level to `DEBUG` is that the debugs may give us a false positive for running tests. IMO this is an issue with output checking in general, which the newer test API tried to address. :\

I think the options are
1) Keep it as `INFO` logs, as they should more accurately reflect the state of Runtime, or
2) Test cases that do output checks should be very specific about what outputs are expected, though this may require using precise regex

Reply:
how could `DEBUG` statements ever cause false positives? if they do then the output check is way too broad

Reply:
One possible way that we could get false positives is if we're checking for something that should appear 5 times but there are DEBUG statements that make it so the output appears more than that? 
I agree that the test cases should ideally not rely on output comparison but I don't really see a way of doing all of that. The test framework overhaul we did at the beginning of the semester went a long way to address that though.

Reply:
idk your example doesn't make sense. why would the debug statements have the output that you are checking for? if that occurs, you can always remove the debug statement since you added it in the first place.

Reply:
We have two functions that do string checks that we currently use for checking different info such as headers for devices or that the "right" error is outputted . Theres one for strings in a certain order vs one for just checking all the strings in general unordered. I believe the example Ben is giving is if we trigger an error two times in a row and then check that an error occurs twice in a row using the add_ordered_string function, we might get a false positive  from the DEBUG statement right after the first error pops up. It'll see that the error string is outputted two times in a row and think the test was successful, but it wouldn't get around to checking the output of the second triggered error since the first error appeared two times, giving us that false positive

Reply:
but why would the debug statement be printing out the same error as another log statement? the debug logs are things you add for testing so you wouldn't choose it to be the same statement. these examples seem very contrived and wouldn't really ever happen